_data/vnext_office_hours.yml

```diff
@@ -12,15 +12,15 @@ extenders:
   - date: 10/20/2023 8AM - 9AM ET
     topic: Explorer (VS Code)
     meeting_link: https://zoom-lfx.platform.linuxfoundation.org/meeting/93849373934?password=15f05b8e-88c9-4bf5-be2b-2a48f3120129
-    recording_link: https://zoom.us/rec/share/J1I8yuRPX-wdEmmNet1PoY5RpmNaVWRzfkn5u0IqYzssLyBxzmsjzroDFLoWQ1lR.J7KEUGMqWML-XQq5
+    recording_link: https://youtu.be/Ur-yPjJyTH8?feature=shared
     materials_link: https://ibm.box.com/s/h5nizhrgke12z63u7v1wivax126usjda
```

Comment:
Could these materials be moved somewhere, like GitHub or the Confluence wiki? Being in an IBM box account probably isn't appropriate here.

Reply:
Well, we haven't figured out yet where to put these things appropriately, I have open issue to figure this out.